src/status_im/ui/screens/syncing/styles.cljs

```diff
@@ -0,0 +1,39 @@
+(ns status-im.ui.screens.syncing.styles
+  (:require [quo2.foundations.colors :as colors]))
+
+(def synced-devices-text
+  {:font-size 13
+   :font-weight :500
+   :color colors/neutral-40})
+
+(def container-main
+  {:margin 16})
+
+(def devices-container
+  {:border-color colors/neutral-20
+   :border-radius 16
+   :border-width 1
+   :margin-top 12})
+
+(def device-row
+  {:flex-direction :row
+   :padding-vertical 10
+   :padding-left 10})
+
+(def device-name
+  {:font-size 15
```

Comment:
you shouldn't use font-size or font-weight, there are props for that in text component